bin/check_lint

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Run code Linter for project packages
+#
+# Usage:
+#> bin/lint [packages ...]
+#
+# Examples:
+#> bin/lint
+#> bin/lint .
+#> bin/lint ./...
+#> bin/lint ./communication
+#> bin/lint ./communication/...
+
+ARGUMENTS=$@
+if [ -z "$ARGUMENTS" ]; then
+    ARGUMENTS=`
+        go list ./... \
+        | grep -v "github.com/mysterium/node/cmd" \
+        | grep -v "github.com/mysterium/node/identity" \
+        | grep -v "github.com/mysterium/node/tequilapi" \
+        | grep -v "github.com/mysterium/node/session" \
+        | grep -v "github.com/mysterium/node/service_discovery/dto" \
+        | grep -v "github.com/mysterium/node/server/dto" \
+        | grep -v "github.com/mysterium/node/openvpn" \
+        | grep -v "github.com/mysterium/node/nat" \
+        | grep -v "github.com/mysterium/node/money" \
+        | grep -v "github.com/mysterium/node/ip" \
+        | grep -v "github.com/mysterium/node/location"
```

Comment:
What does this list mean? Can we add a comment description, or put this list in a variable with clear name?

Reply:
Decision is:
1. We care about quality and run CI checks for new code
2. We declare packages which are not-clared and clear them in long-run 
@donce @zolia @tadovas

Reply:
Awesome idea. But I would like to have this clear in separate variable like "packagesNeedToClean" or something. Because now I need to look very closely if they are blacklist or whitelist.

Reply:
Improved